supervisor/api/__init__.py

```diff
@@ -228,12 +228,21 @@ def _register_hardware(self) -> None:
             ]
         )
 
-    def _register_info(self) -> None:
-        """Register info functions."""
-        api_info = APIInfo()
-        api_info.coresys = self.coresys
+    def _register_root(self) -> None:
+        """Register root functions."""
+        api_root = APIRoot()
+        api_root.coresys = self.coresys
+
+        self.webapp.add_routes([web.get("/info", api_root.info)])
+        self.webapp.add_routes([web.post("/refresh_updates", api_root.refresh_updates)])
+        self.webapp.add_routes(
+            [web.get("/available_updates", api_root.available_updates)]
+        )
 
-        self.webapp.add_routes([web.get("/info", api_info.info)])
+        # Remove 2023
```

Comment:
I think this is too early, if anyone is on 2021.12.x when this is removed would need to use CLI to get to the update.